src/mono/msbuild/common/MonoSharedBuild.props

```diff
@@ -0,0 +1,23 @@
+<Project>
```

Comment:
When introducing a new props or targets file, you'll need to make sure it gets distributed in https://github.com/dotnet/runtime/blob/d40f90c3b05570d40da2bb12c859d6466451c8ef/src/mono/nuget/Microsoft.NET.Runtime.LibraryBuilder.Sdk/Microsoft.NET.Runtime.LibraryBuilder.Sdk.pkgproj#L8-L19

Reply:
Additionally, you should add the common path property probably here https://github.com/dotnet/runtime/blob/main/src/mono/nuget/Microsoft.NET.Runtime.LibraryBuilder.Sdk/Sdk/Sdk.props

Reply:
Thank you, do we have some tests that would verify this for the library builder, or we ensure this manually by unzipping the generated nuget?

Reply:
Fixed with: https://github.com/dotnet/runtime/pull/91800/commits/0ec903efb2e546758d7e130e3eb3d4b8641103cb